lib/chef/compliance/default_attributes.rb

```diff
@@ -39,7 +39,7 @@ module Compliance
       "insecure" => nil,
 
       # Controls verbosity of Chef InSpec runner. See less output when true.
```

Comment:
please update the comment to signify proper function of this change

Reply:
thanks. done.